README.md

```diff
@@ -77,6 +77,9 @@ Check Spelling inside identifiers
 "ignoreRequire": <<Boolean>>, default: false
 Exclude `require()` imports from spell-checking. Useful for excluding NPM package name false-positives.
 
+"ignoreUpperCaseUnderscore": <<Boolean>>, default: false
```

Comment:
meybe we need to set this to false to be retrocompatible

Reply:
`ignoreUpperCaseUnderscore` default value is `false` now, I guess your means is to set the`ignoreUpperCaseUnderscore` default value to true, then the behavior will be retro compatible right?